app/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_status_validation_mixin.py

```diff
@@ -25,7 +26,8 @@ def test_update_experiment_publish_status_while_in_preview(self):
             },
             context={"user": self.user},
         )
-        self.assertTrue(serializer.is_valid())
+        self.assertFalse(serializer.is_valid())
+        self.assertIn("publish_status", serializer.errors)
```

Comment:
Interesting this is another one of those cases where we allowed a transition that doesn't actually make sense.  There is no publish flow for moving into or out of preview so you shouldn't be able to update publish status while in preview.  Good catch 👍